server/lib/search.js

```diff
@@ -151,17 +156,17 @@ export const searchCollectivesInDB = async (
   }
 
   // Cleanup term
+  term = sanitizeSearchTermForILike(trimSearchTerm(term));
   if (term && term.length > 0) {
-    term = sanitizeSearchTermForILike(trimSearchTerm(term));
     if (term[0] === '@') {
```

Comment:
Why are we getting that `term = ` out of the `if (term && term.length > 0) {`? I don't see the goal of saniziting if it's empty, and it will crash if term is nullish.

Reply:
Corrected this one as well. 👍🏽